### PR TITLE
Issue 7 box request review form tags

### LINKED ISSUE #7: BoxRequest Review Form (tags)
Box Request and Requester data should display, along with the extra fields: tags, and summary

### PR DESCRIPTION
**Pull request by gabezurita**
Resolves #7

### Description
Adds box review functionality to the app in the following view:
http://localhost:5000/box_requests

Users can now review box requests, which includes adding a summary and tags to the box request.

Users without the `REQUEST_REVIEWER` permission cannot view or edit box requests.

### Future Work
We still need to add styling and work on the way users add tags (front-end changes)

Follow on change issue here: https://github.com/rubyforgood/voices-of-consent/issues/141

### Type of change
* New feature (non-breaking change which adds functionality)

### Screenshots

image

### CODE REVIEW

File: app/controllers/box_requests_controller.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class BoxRequestsController < ApplicationController
-  before_action :set_box_request, only: [:show, :edit, :update, :destroy]
+  before_action :set_box_request, only: %i[show edit update destroy]
```
Comment (h-m-m) [inline app/controllers/box_requests_controller.rb:2]: I appreciate all the small cleanups you've added in this PR. I won't comment on every one other than to say thanks.

File: app/views/box_requests/show.json.jbuilder [hunk 2]
```diff
@@ -1 +1 @@
-json.partial! "box_requests/box_request", box_request: @box_request
+json.partial! "box_request_reviews/box_request_review", box_request: @box_request
```
Comment (h-m-m) [inline app/views/box_requests/show.json.jbuilder]: Is this a partial that needs to be added? I don't see it already on the develop branch.

Comment (maebeale) [reply app/views/box_requests/show.json.jbuilder]: @gabezurita hey! do you know the answer to @h-m-m 's q above?

Comment (gabezurita) [reply app/views/box_requests/show.json.jbuilder]: Hi Mae! So sorry for the slow reply, I've been traveling through Chile and will not be back to the US until August 19. Is my input still needed here? Seem as if you've picked up the ball?

I can go back to working on issues once I'm back to the USA!

Comment (maebeale) [reply app/views/box_requests/show.json.jbuilder]: @gabezurita hey there! NO problem at all! yes, i figured it out and handled this thread. enjoy chile! we'll be here when you get back. :) thank you for everything!

**Pull request merged**